tests/test_modeling_common.py

```diff
@@ -3224,7 +3224,6 @@ def test_flash_attn_2_fp32_ln(self):
     def test_tf_from_pt_safetensors(self):
         for model_class in self.all_model_classes:
             config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
-            config.tie_word_embeddings = True  # Tied weights often cause safetensors loading to fail
```

Comment:
Why remove this?

Reply:
I added it in this PR to check if it caused any more failures - it didn't, so I took it back out again. It doesn't really fit in this test anyway, I just wanted to see if there were more models that might have issues with it!